Add hook at micro frontend container creation (#894)
SAP/luigi: docs/general-settings.md

```
> **NOTE:** You can translate Luigi internal messages by providing translation for [these keys](../core/src/utilities/defaultLuigiTranslationTable.js).
* **customSandboxRules** is an array of custom rules for the content in the iframe. You can extend the [Luigi default sandbox rules](https://github.com/SAP/luigi/blob/af1deebb392dcec6490f72576e32eb5853a894bc/core/src/utilities/helpers/iframe-helpers.js#L140) by adding further rules.
* **iframeCreationInterceptor** is a function called on iframe creation. It gives you full control over the created iframe DOM element. You can modify it to your needs just before it is added to the DOM tree. 
This function is called with these parameters: 

>   * `iframe` the iframe DOM element, not yet added to the DOM tree but all attributes are set already

Reviewer: ```suggestion
  * `iframe` is the iframe DOM element. It is not yet added to the DOM tree, but all attributes are already set.
```

>   * `viewGroup` the view group associated with this iframe, if any

Reviewer: ```suggestion
  * `viewGroup` is the view group associated with this iframe, if any.
```

>   * `microFrontendType` which is 'main', 'modal' or 'split-view' depending on where it is going to be rendered

Reviewer: ```suggestion
  * `microFrontendType`, which is `main`, `modal` or `split-view` depending on where it is going to be rendered.
```

```
> **NOTE:** You can translate Luigi internal messages by providing translation for [these keys](../core/src/utilities/defaultLuigiTranslationTable.js).
* **customSandboxRules** is an array of custom rules for the content in the iframe. You can extend the [Luigi default sandbox rules](https://github.com/SAP/luigi/blob/af1deebb392dcec6490f72576e32eb5853a894bc/core/src/utilities/helpers/iframe-helpers.js#L140) by adding further rules.
* **iframeCreationInterceptor** is a function called on iframe creation. It gives you full control over the created iframe DOM element. You can modify it to your needs just before it is added to the DOM tree. 
This function is called with these parameters: 
  * `iframe` is the iframe DOM element. It is not yet added to the DOM tree, but all attributes are already set.
  * `viewGroup` is the view group associated with this iframe, if any.

>   * `microFrontendType`, which is `main`, `modal` or `split-view` depending on where it is going to be rendered.

Reviewer 2: I would add here a snippet as an example of how you can use it.

Author: good idea, done!

Author: done!

```
> **NOTE:** You can translate Luigi internal messages by providing translation for [these keys](../core/src/utilities/defaultLuigiTranslationTable.js).
* **customSandboxRules** is an array of custom rules for the content in the iframe. You can extend the [Luigi default sandbox rules](https://github.com/SAP/luigi/blob/af1deebb392dcec6490f72576e32eb5853a894bc/core/src/utilities/helpers/iframe-helpers.js#L140) by adding further rules.
* **iframeCreationInterceptor** is a function called on iframe creation. It gives you full control over the created iframe DOM element. You can modify it to your needs just before it is added to the DOM tree. 
This function is called with these parameters: 
  * `iframe` is the iframe DOM element. It is not yet added to the DOM tree, but all attributes are already set.
  * `viewGroup` is the view group associated with this iframe, if applicable.
  * `navigationNode` is the navigation node associated with this iframe. NOTE: the interceptor is called only once per iframe creation. If two or more navigation nodes share the same iframe (because they belong to the same view group) the interceptor is called with the first navigated node only. 

>   * `microFrontendType`, which is `main`, `modal` or `split-view` depending on where it is going to be rendered.

Reviewer: ```suggestion
  * `microFrontendType`, which is `main`, `modal` or `split-view` depending on where it is going to be rendered.
  
  For example, to allow 'fullscreen' for non-modal iframes:
```

```javascript
{

>   // for example, allow 'fullscreen' for non-modal iframes:

Reviewer: ```suggestion
```
